### ISSUE TITLE
Views metadata table including which collection ids are covered by which views

### ISSUE THREAD

**Issue by pindec** [open-contracting/kingfisher-colab#44]
Would be helpful to have a metadata views table in Kingfisher, auto populated by views creation, to map collection ids to view names.

Also helpful for this table to store who created it (name of analyst), when, expiration date if known and any other notes (instead of looking in each view for notes).

Use case 1: for any given collection, we can query what views have been generated on it ordered by creation date desc

Use case 2: set up a cron job to delete the views as expiration dates pass. 

Use case 3: cross-collection queries - for each view, if we haven't already run a query on the collection ids it covers, and the expiration date hasn't passed, run a query on it.

**Comment by jpmckinney** [comment]
The function should use `schema` not `views`, because "views" is just confusing and is not used anywhere anymore.

**Comment by duncandewhurst** [comment]
As Charlie mentioned, our main use case for this is in Colaboratory. However, when I need to check something quickly, I sometimes connect directly to the Kingfisher database from a desktop query tool. I understand that other helpdesk analysts do this too. Therefore I think it would be better if we store this information in a table in the database, then it is accessible however a user connects.

**Comment by pindec** [comment]
Yes, this info in a table would be helpful for quick data checks via my standalone query tool.

**Comment by jpmckinney** [comment]
Right now, Summarize can create new schema, and can only create/update tables in those schema. To add this feature, we'd need to change Summarize to:

* Create a schema for metadata (e.g. `summaries`) and a table for this issue's metadata (`e.g. selected_collections`), if either of those doesn't exist
* In `add`, add rows to the table (creating the table/schema as above if it doesn't exist – we don't need a separate `install` command)
* In `remove`, delete rows from the table

I assume the table would have the columns `(id SERIAL, schema TEXT NOT NULL, collection_id INTEGER NOT NULL)`.

We should get rid of the `selected_collections` table in each schema, as it's otherwise duplication.

If that sounds good, I can move the issue back to Summarize.

**Comment by duncandewhurst** [comment]
Sounds good to me!
